frontend/components/SendToGravePanel.tsx

```diff
@@ -0,0 +1,339 @@
+      setTokenCheckMessage('');
+      setShowGhosts(true);
+      setTimeout(() => setShowGhosts(false), 4000);
+      toast({
+        title: `Gone but not forgotten! See you in the afterlife.`,
+        status: 'success',
+        position: 'bottom-left',
+        duration: 9000,
+        isClosable: true,
+      });
+    } catch (error: any) {
+      setIsSubmitting(false);
+      setTokenCheckMessage('');
+      console.error(error);
+      toast({
+        title: `Error sending token to Grave. ${error.message}`,
+        status: 'error',
+        position: 'bottom-left',
+        duration: 9000,
+        isClosable: true,
+      });
+    }
+  };
+
+  const FieldMessage = () => {
+    if (tokenCheckMessage) {
+      return tokenCheckMessage;
+    }
+    return null;
+  };
+
+  const renderGhosts = () => {
+    return Array.from({ length: 10 }).map((_, index) => (
+      <div
+        key={index}
+        className="ghost"
+        style={{
+          left: `${Math.random() * window.innerWidth}px`,
+          top: `${Math.random() * window.innerHeight}px`,
+        }}
+      >
+        👻
+      </div>
+    ));
+  };
+
+  return (
+    <>
+      <Text
+        fontSize="20px"
+        fontWeight="bold"
+        fontFamily="Bungee"
+        color="dark.purple.400"
+      >
+        SEND ASSETS TO YOUR GRAVE
+      </Text>
+      <Text
+        mb={4}
+        mt={4}
+        fontWeight={600}
+        fontFamily="Montserrat"
+        textAlign="start"
+      >
+        Have some trash in your 🆙? Send it to the GRAVE! Whether it's NFTs or
+        tokens, click "POOF!" and let them rest in peace!
+      </Text>
+      <FormControl textAlign="start">
+        <FormLabel fontFamily="Bungee" fontWeight={400} fontSize={'14px'}>
+          ASSET ADDRESS
+        </FormLabel>
+        <Box display="flex">
+          <Flex flexDir="column">
+            <Input
+              autoComplete="off"
+              maxWidth="314px"
+              minWidth="200px"
+              height="25px"
+              fontFamily={'Bungee'}
+              backgroundColor="white"
+              borderColor={'var(--chakra-colors-dark-purple-500)'}
+              _hover={{
+                borderColor: 'var(--chakra-colors-dark-purple-500)',
+              }}
+              _focus={{
+                borderColor: 'var(--chakra-colors-dark-purple-500)',
+                boxShadow: 'none',
+              }}
+              value={inputTokenAddress}
+              onChange={handleChange}
+              placeholder="PASTE ASSET ADDRESS"
+              _placeholder={{
+                fontWeight: 'bold',
+                color: 'var(--chakra-colors-dark-purple-200)',
+              }}
+            />
+            <Text ml={2} fontFamily="Bungee" fontWeight={400} fontSize={'14px'}>
+              {FieldMessage()}
+            </Text>
+          </Flex>
+        </Box>
+      </FormControl>
+      <Flex justifyContent={'flex-start'} gap={2} width={'100%'}>
+        <Button
```

Comment:
Should this have a loadingText={FieldMessage()}? 

That way we remove the text field above and our button will automatically change text based on the status, as that's what we do with other buttons like revive

Reply:
@democrazyfi those messages are much more general than just loading states. although in general I agree we should standardize this, we can do that during a larger codebase refactor down the line